app/initializers/session.js

```diff
@@ -0,0 +1,21 @@
+export function initialize(container, application) {
+    application.deferReadiness();
+
+
+    var session = container.lookup('controller:session');
```

Comment:
@icereval @chrisseto 
based on http://emberjs.com/deprecations/v1.x/,
seems like this will have to change for 2.x compliance.

Trying to wrap my head around how session works, and failing. Any thoughts?